ironfish/src/consensus/verifier.test.ts

```diff
@@ -55,7 +55,7 @@ describe('Verifier', () => {
       )
 
       expect(result).toEqual({
-        reason: VerificationResultReason.ERROR,
+        reason: VerificationResultReason.MINIMUM_FEE_NOT_MET,
```

Comment:
Miners fee transactions were originally being caught by pool.verify. We have replaced this functionality with the min fee check, so we get an even better error message